src/containers/WhiteContainer.tsx

```diff
@@ -0,0 +1,29 @@
+import { Grid } from "@mui/material";
+import FormStepper from "components/Form/FormStepper";
+import { IState } from "components/Form/types";
+import ButtonContainer from "components/Form/FormButton/ButtonContainer";
+
+interface IProps {
+  step: number;
+  children: JSX.Element | JSX.Element[];
```

Comment:
children: JSX.Element, {children} will crush if you pass list. You can pass one nested children.